jetbrains-core/tst/software/aws/toolkits/jetbrains/services/lambda/upload/CreateLambdaFunctionTest.kt

```diff
@@ -53,36 +54,37 @@ Resources:
       Handler: helloworld.App::handleRequest
       Runtime: foo
       Timeout: 800
-""")
+"""
+        )
     }
 
-    @Test
+    @Test(expected = IllegalArgumentException::class)
```

Comment:
Any reason to use this vs `assertThatThrownBy`

Reply:
We seem to use expected a lot. Beyond that, not really. Will change to use `assertThatThrownBy`

Reply:
Maybe we should ban expected too :D